ticks/graphql-api.txt

```diff
@@ -0,0 +1,4 @@
+module "GRAPHQL_API_MIX_PATH/GraphQL.Internal.Syntax.AST" {
```

Comment:
Yes, it is an overlay of hpc.  I did it because `position` is not intended to be tested as function.  (Haskell will translate records@{position = x}` to `records _ x`, and therefore is not considered tested in HPC) 

One method is to add one in hpc python script, but I think it makes more sense to use a ticks file to explicitly tell those functions we do not need to test.